### ISSUE TITLE
Office add-in not updating automatically

### ISSUE THREAD

**Issue by ThoughtRiverAdmin** [OfficeDev/office-js#1655]
I submitted a new manifest file for an Office add-in on AppSource and the change was approved by the AppSource team, however, users of the add-in were still able to use the old version of the manifest file. This meant they were using an outdated version of the add-in that no longer worked as intended.

My question is should Microsoft Word automatically update add-ins when there is a new version on the Office Store/AppSource? In the past I have been prompted to update the add-in if there is a new version on the Office Store/AppSource, so does this mean there are some instances were the update prompt may not appear, or can the prompt be ignored thus allowing a user to continue using the old version?

**Comment by lliu113** [comment]
@ThoughtRiverAdmin  - as you mentioned, your users should see a prompt to update the next time they use the add-in. The prompt cannot be ignored - users should not be able to use an older version of your manifest if a new one is available.

It may take a little time for the new manifest info to propagate, how long are your users still able to use the older manifest for?

**Comment by ThoughtRiverAdmin** [comment]
@lliu113 It was a few days after the new add-in version was showing on the Office Store/AppSource. Since the previous manifest pointed towards a service that's no longer used, it caused the user to no longer be able to use the add-in. We were able to solve it by getting the user to re-download the add-in from the store.

Do you know if the old manifest file is cleared from the Office cache once an add-in update is detected? Could this potentially be impacted by an organization's security policy settings?

**Comment by jordandev** [comment]
We are seeing a similar problem. Have version 1.0.0.1 of our add-in installed. 1.0.0.2 is live on the store. But never getting a prompt to update (when using the add-in after restarting Excel, etc).

Back when we had deployed 1.0.0.1 a couple months ago I did get the prompt to update from 1.0.0.0 as expected. But another user didn't.

So something appears to inconsistently not work here. This is on Excel for Windows.

Note that Excel Online did show the prompt for 1.0.0.2 as expected right away trying to use the add-in.
